packages/iov-bcp/types/connection.d.ts

```diff
@@ -194,4 +194,5 @@ export interface BcpConnection {
      * the query, along with all new transactions arriving from listenTx
      */
     readonly liveTx: (txQuery: BcpTxQuery) => Stream<ConfirmedTransaction | FailedTransaction>;
+    readonly getFeeQuote: (tx: UnsignedTransaction) => Promise<Fee | null>;
```

Comment:
Good choice using a promise. This allows us to query data from the network, like this one: https://web3js.readthedocs.io/en/1.0/web3-eth.html#getgasprice

However, I don't think we need the `null` case. In case of BNS, we can return a zero amount or even better, change to a small anti-spam fee